Cargo.toml

```
@@ -60,6 +60,7 @@ path = "src/lib.rs"
 external-secp = []
 
 [dependencies]
+bitflags = "2.5.0"
```

Ordinarily we don't pin past `major.minor` unless needed to set a lower bound.
```suggestion
bitflags = "2.5"
```